src/components/OrgView/OrgView.js

```diff
@@ -137,8 +138,14 @@ function OrgView({
         >
           <div
             css={`
-              height: 100%;
               display: flex;
+              height: 100%;
+              ${iOS || isSafari
+                ? `
+                  height: 100vh;
```

Comment:
I’m wondering if using `flex-grow: 1` wouldn’t have the same effect on Safari? (I don’t have a macOS to test right now).

Also I am not sure what it’s for, but we should probably explain what the `padding-bottom` is doing as a comment.

Reply:
Nope, `flex-grow: 1` is not working.

Reply:
Oh of course sorry, the parent is not flex. Does it work if we make it so?

Reply:
Nope. Added `display: flex` to parent container and `flex-grow: 1` to element and it was not sizing up.

Reply:
Wonder if a `calc(100vh - 8 * GU)` would work here, to avoid the bottom padding?

Reply:
I think I did try that but with a `+`, I can give another go just to add to your peace of mind but I did test out different options before having to go for the os/browser specific solution.

Reply:
Huh? It worked but why? Could you help me understand why removing the value from the height gives it the correct height? I would initially think it would make it shorted...

Reply:
The old solution was a container with 100vh + 8GU bottom padding. Essentially:

```
|     |
|-----| <--- screen end
|     |
| 8GU |
|     |
|-----| <--- 100 vh (container end); from the screen's end, this bit would still get cut off
```

Now, instead, what we have is a container whose height is 100vh - 8GU:


```
|     |
|-----| <--- screen end and container end
|     |
| 8GU |
|     |
|-----| <--- 100vh
```

I didn't like the padding before because anything that relied on that container's height to place it at the bottom (e.g. a footer) would still get pushed out of the screen. Instead, what we've done now is tell the container to take what's remaining of the 100vh (due to the fixed top bar).